odl/discr/tensor_ops.py

```diff
@@ -669,6 +669,67 @@ def adjoint(self):
                               weight=self.weights)
 
 
+# TODO: Optimize this to an optimized operator on its own.
+class PointwiseSum(PointwiseInner):
+
+    """Take the point-wise sum of a vector field.
+
+    This operator takes the (weighted) sum
+
+        ``sum(F(x)) = [ sum_j( w_j * F_j(x) ) ]
+
+    where ``F`` is a vector field. This implies that
```

Comment:
No objections against "vector field"? Here it wouldn't be a big pain to change to "vector-valued function" which is a more generous term. The variable name could even stay :-)